rgb2cry.c

```diff
@@ -1,27 +1,37 @@
 #include <stdio.h>
+#include <stdlib.h>
 
 typedef unsigned short UWORD;
 typedef unsigned char UBYTE;
 typedef short WORD;
 typedef char BYTE;
 
 UWORD do_cry(int, int, int);
-extern int atoi(char *);
+//extern int atoi(char *);
```

Comment:
Just delete it rather than commenting it out.  It's still there in git history if someone needs to dig it up.

Reply:
This is good point, fixed.